lookout/style/format/tests/test_features.py

```diff
@@ -25,6 +25,15 @@ def setUpClass(cls):
             cls.uast = bblfsh.Node.FromString(fin.read())
         cls.extractor = FeatureExtractor("javascript", parents_depth=2, siblings_window=5)
 
+    def test_parse_file2(self):
+        test_js_code_filepath = Path(__file__).parent / "for_parse_test.js.xz"
+        with lzma.open(str(test_js_code_filepath), mode="rt") as f:
+            code = f.read()
+        uast = bblfsh.BblfshClient("0.0.0.0:9432").parse(
+            filename="", language="javascript", contents=code.encode()).uast
```

Comment:
no need to specify filename when contents is set. It could also be good to just bypass babelfish and just use a saved uast as we do in the other tests. Up to you :)

Reply:
nope. here the header of parse function: `def parse(self, filename, language=None, contents=None, timeout=None):` also the file name goes to bblfsh logs.
When we update to the next version of a bblfsh driver, we have to update this files.
I prefer to have it like that.

Reply:
Fair enough, even though we will have to update all the other files in the folder anyway. And huh, this function signature needs refactoring, I'll open an issue.